README.md

```diff
@@ -16,12 +16,13 @@ Before setting up OpenEBS CSI driver make sure your Kubernetes Cluster
 meets the following prerequisites:
 
 1. You will need to have Kubernetes version 1.14 or higher
-2. You will need to have OpenEBS Version 1.1 or higher installed. 
+2. You will need to have OpenEBS Version 1.2 or higher installed. 
    The steps to install OpenEBS are [here](https://docs.openebs.io/docs/next/quickstart.html)
 3. iSCSI initiator utils installed on all the worker nodes
 4. You have access to install RBAC components into kube-system namespace.
    The OpenEBS CSI driver components are installed in kube-system 
-   namespace to allow them to be flagged as system critical components. 
+   namespace to allow them to be flagged as system critical components.
+5. You will need to turn on  ExpandCSIVolumes and ExpandInUsePersistentVolumes feature gates on  kubelets and kube-apiserver:
```

Comment:
Are there GA/Beta in 1.15 K8s?

Reply:
ExpandInUsePersistentVolumes has been moved to Beta while ExpandCSIVolumes is still in alpha stage as of 1.15.